site/docs/layer.md

```diff
@@ -5,5 +5,35 @@ title: Layering views
 permalink: /docs/layer.html
 ---
 
-TODO
+{: .suppress-error}
+```json
+{
+  "$schema": ...,
+  "layer": [ // Layer array
+    ... // - Chart Objects
+  ]
+}
+```
 
+Sometimes, it's useful to superimpose one chart on top of another. You can accomplish this by using the `layer` property.
+Defining a layered chart is simple. You only have to assign an array of chart objects to `layer`.
```

Comment:
Let's avoid saying that something is simple. It implies that if it doesn't work, that it is the fault of the user.